adapter_darwin.go

```diff
@@ -100,6 +101,18 @@ func (cmd *centralManagerDelegate) DidDiscoverPeripheral(cmgr cbgo.CentralManage
 	}
 }
 
+// DidDisconnectPeripheral when peripheral is disconnected.
+func (cmd *centralManagerDelegate) DidDisconnectPeripheral(cmgr cbgo.CentralManager, prph cbgo.Peripheral, err error) {
+	if err != nil {
+		fmt.Println(err.Error())
```

Comment:
I am not sure of the correct way to handle this error here. Ideally, the callback would be able to accept it so we can surface it to the user in a meaningful way.

Reply:
Sweet, okay. Happy to just ignore it for now then. 

It would be good to have some mechanism in the future to expose it, but as the availability and type of the error are going to be platform dependent, it doesn't make it simple. Something to explore in a later PR.